.changeset/smooth-ways-retire.md

```diff
@@ -0,0 +1,24 @@
+---
+'@primer/react-brand': minor
+---
+
+Added a new `FrostedGlassVFX` component for applying a frosted glass-effect texture to nested components.
+
+⚠️ This is an experimental component, and not tested for compatibility with all existing Primer Brand components.
+
+Usage example:
+
+```jsx
+<FrostedGlassVFX>
+  <Testimonial variant="default">
+    <Testimonial.Quote>
+      GitHub helps us ensure that we have our security controls baked into our pipelines all the way from the first line
+      of code we&apos;re writing.
+    </Testimonial.Quote>
+    <Testimonial.Name />
+    <Testimonial.Avatar />
+  </Testimonial>
+</FrostedGlassVFX>
+```
+
+🔗 (See Storybook for an example)[https://primer.style/brand/storybook/?path=/story/components-testimonial-examples--with-frosted-glass]
```

Comment:
```suggestion
🔗 (See Storybook for an example)[https://primer.style/brand/storybook/?path=/story/components-testimonial-examples--with-frosted-glass]

⚠️  When using this component, ensure sufficient contrast between text and the background behind `FrostedGlassVFX`. The frosted glass effect can reduce contrast, especially:
- In light mode: Dark background areas may make dark text hard to read
- In dark mode: Light background areas may make light text hard to read

Test contrast across all viewport sizes when using `FrostedGlassVFX` over varied or multicoloured backgrounds.
```

Reply:
Thanks @joshfarrant - I've adapted your suggestion and put it earlier in the log entry.